frontend/src/components/Chat/ChatInput.tsx

```diff
@@ -129,15 +128,15 @@ function ChatInput({ messageType, directMessageId, isDisabled }: ChatInputProps)
             placeholder='Send a message...'
             borderRadius='0'
             className='text-area'
-            disabled={isDisabled}
+            disabled={false}

```

Comment:
Are you able to delete this line entirely? Or does TextArea not have a default disabled behavior?

Reply:
hmm thought I tried before but now it's fine with a default false behavior :) - removed now